omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+<!--
+
+author:   Meredith Lee
+email:    leemc@chop.edu
+version:  1.0.0
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Omics Orientation
+comment: This module provides a brief introduction to omics and its associated fields.
+long_description: Omics is a wide-reaching field, with many different subfields. This module aims to disambiguate several omics-related terms and topics, discuss some of the most popular omics research fields, and examine the challenges of and caveats for omics research.
+estimated_time: 15m
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Define what omics is and explain why a researcher might choose an omics approach
+- Identify several popular omics domains
+- Describe some challenges and caveats of omics research
+
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Omics Orientation
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** None
+
+**Learning Objectives**
+
+@learning_objectives
+
+
+</div>
+
+
+## What is "Omics"?
+
+<div class = "learnmore">
+Much of the content of this module, particularly the definitions of specific omics domains, was derived from an article by Hasin *et al.* published in the journal *Genome Biology*. The citation is below and the article is open source if you are interesting in reading more about this topic.
+
+<div style = "margin-left: 40px; text-indent: -40px; font-size:0.8em;">
+
+Hasin, Y., Seldin, M. & Lusis, A. Multi-omics approaches to disease. Genome Biol 18, 83 (2017). https://doi.org/10.1186/s13059-017-1215-1
+
+</div>
+
+</div>
+
+**Omics** is a suffix added to a large number of biological disciplines which denotes a "comprehensive, or global, assessment of a set of molecules" (Hasin et al., 2017) related to that domain.  Genomics is possibly the most well-known member of this family of disciplines, but is certainly not the only one! Omics has become something of a buzzword in recent years with the improvement of high-throughput technologies and increased ability to analyze large amounts of data.
```

Comment:
Maybe a pronunciation guide to the word "omics," although I'm not totally sure that is necessary.

Reply:
Mmm yeah that makes sense, doesn't hurt to add it.